blazor/diagram/symbol-palette.md

```diff
@@ -821,6 +822,85 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ![Tooltip in symbol palette](images/differenttooltip.gif)
 
+
+### Tooltip template for symbols
+
+You can provide custom template as tooltip for symbols in the symbol palette using SymbolTooltipTemplate. Once tooltip template is defined then enable the custom tooltip for symbols in the symbol palette by setting the Tooltip constraints for node and connector. This allows the tooltips template to be displayed when hovering over symbols in the symbol palette.
```

Comment:
You can provide custom template as tooltip for symbols in the symbol palette using `SymbolTooltipTemplate` property of `SfDiagramComponent`. Once tooltip template is defined then enable the custom tooltip for symbols in the symbol palette by setting the Tooltip constraints for node and connector. This allows the tooltips template to be displayed when hovering over symbols in the symbol palette.